xl_auth/templates/users/edit_details.html

```diff
@@ -14,14 +14,6 @@
                 {{ edit_details_form.full_name.label }}
                 {{ edit_details_form.full_name(class_="form-control") }}
             </div>
-            <div class="form-group">
```

Comment:
This is another "premature optimization", since I thought that we might want to expand the User model with stuff that regular users may edit in the future. "Details" is vague enough that it might include anything, while `edit_*_name` theoretically would require another name change if we decide to expand.